WordPress/src/main/java/org/wordpress/android/ui/themes/ThemeActivationBottomSheetFragment.kt

```diff
@@ -0,0 +1,37 @@
+package org.wordpress.android.ui.themes
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.View.INVISIBLE
+import android.view.View.VISIBLE
+import android.view.ViewGroup
+import androidx.annotation.NonNull
+import com.google.android.material.bottomsheet.BottomSheetDialogFragment
+import org.wordpress.android.R
+import org.wordpress.android.databinding.ThemeActivationBottomSheetBinding
+
+class ThemeActivationBottomSheetFragment : BottomSheetDialogFragment() {
```

Comment:
It would be nice if this `ThemeActivationBottomSheetFragment` was launched when the user presses the **Activate** button. Even if the feature is not fully testable end-to-end and you plan to refactor the architecture in a next PR this will allow us to test the UI and receive feedback from the design team.